### PR TITLE
Fixes #16518: Missing report on "Synchronize files" on simple relays when shared-files are empty

### PR DESCRIPTION
**Pull request by amousset**
https://issues.rudder.io/issues/16518

### CODE REVIEW

File: techniques/system/distributePolicy/1.0/propagatePromises.cf [hunk 1]
```diff
@@ -158,7 +158,8 @@ bundle agent propagatePromises
 
     pass3.files_propagated.sync_method_rsync::
       "any" usebundle => rudder_common_report("DistributePolicy", "result_success", "${distribute_policy_common.directiveId}", "Synchronize files", "None", "Shared files synchronized using rsync");
-    pass3.(files_propagated|(could_not_propagate_files.update_successful)).!sync_method_rsync::
+    # Cannot directly use seccess class as it is not defined if there are no files to copy
```
Comment (VinceMacBuche) [inline techniques/system/distributePolicy/1.0/propagatePromises.cf]: seccess => success

**Comment by amousset** [comment]
PR updated with a new commit

File: techniques/system/distributePolicy/1.0/propagatePromises.cf (hunk 1 above)
Comment (amousset) [reply techniques/system/distributePolicy/1.0/propagatePromises.cf]: :eye:

**Comment by amousset** [comment]
PR updated with a new commit

**Pull request closed without merging**